hugo/layouts/partials/home/roadmap.html

```diff
@@ -1,27 +1,24 @@
 <section class="roadmap">
-    <div>
+    <article>
         <img class="code-pills-left" src="{{ "img/bgs/codepills-bg-3.png" | absURL }}" />
 
-        <article>
-            <div class="roadmap__photo">
-                <img src="{{ .roadmap.ceo_picture | relURL }}" alt="Eiso Kant" />
-            </div>
-
-            <div class="roadmap__content">
-                <h1>Our roadmap</h1>
-                {{ range $r := .roadmap.content }}
-                    <p>{{ $r }}</p>
-                {{ end }}
+        <h1>Our roadmap</h1>
+        {{ range $r := .roadmap.content }}
+            <p>{{ $r }}</p>
+        {{ end }}
 
-                <div class="roadmap__signature">
-                    <p class="roadmap__signature__name">Eiso Kant</p>
-                    <p>Co-Founder &amp; Ceo</p>
+        <footer class="roadmap__footer">
+            <div class="roadmap__signature">
+                <img class="roadmap__signature__photo" src="{{ .roadmap.ceo_picture | relURL }}" alt="" />
```

Comment:
Eiso's picture is now quite smaller than in the previous version. Just checked the documentation and it had 180px (outdated), so fixed it there to 90px to match the PSD/doc screenshot.